docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md

```diff
@@ -0,0 +1,104 @@
+# Dasharo Compatibility: USB Boot
+
+## Test cases
+
+### UBB001.001 USB detect and boot after coldboot
+
+**Test description**
+
+This test aims to verify that the DUT properly detects USB device and boots into
+operating system after coldboot.
+
+**Test configuration data**
+
+1. `BOOT_MENU_KEY` = `ESC`
+2. `BOOT_MENU_STRING` = `Press ESC for boot menu`
+3. `USB_STICK` = `USB MSC Drive General USB Flash Disk 1100`
+
+**Test setup**
+
+1. Proceed with the [Generic test setup: firmware](generic-test-setup.md#firmware).
+
+**Test steps**
+
+1. Disconnect the DUT from the power source and then wait a few seconds.
+2. Power on the DUT.
+3. Wait for boot until `BOOT_MENU_STRING` appears.
+4. Press `BOOT_MENU_KEY` to enter the boot menu.
+5. Check if `USB_STICK` entry is available.
+6. Select the proper number for `USB_STICK` option.
+7. Wait for `Debian GNU/Linux`.
+
+**Expected result**
+
+The `Debian GNU/Linux` is visible and confirms successful boot into OS after
+coldboot.
+
+### UBB001.002 USB detect and boot after warmboot
+
+**Test description**
+
+This test aims to verify that the DUT properly detects USB device and boots into
```

Comment:
In the test case desription we should add information, that the test case may be re-done several times to specify the platform and connection stability.